projects/Mallard/src/screens/settings/join-beta-button.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import { Heading } from 'src/components/layout/ui/row'
+import { List } from 'src/components/lists/list'
+import { Linking } from 'react-native'
+import { JOIN_BETA_LINK } from 'src/constants'
+import { isInBeta } from 'src/helpers/release-stream'
+import remoteConfig from '@react-native-firebase/remote-config'
+
+const betaThanks = () => (
+    <>
+        <Heading>{``}</Heading>
+        <Heading>Thank you for being a beta tester 🙌</Heading>
+        <Heading>{``}</Heading>
+    </>
+)
+
+const joinBetaMenuButton = () => (
+    <>
+        <Heading>{``}</Heading>
+        <List
+            data={[
+                {
+                    key: 'Become a beta tester 🙌',
+                    title: 'Become a beta tester 🙌',
+                    onPress: () => {
+                        Linking.openURL(JOIN_BETA_LINK) //what to catch here?
+                    },
+                },
+            ]}
+        />
+        <Heading>{``}</Heading>
+    </>
+)
+
+const BetaButtonOption = () => {
+    if (!!remoteConfig().getValue('join_beta_button_enabled').value) {
+        return isInBeta() ? betaThanks() : joinBetaMenuButton()
```

Comment:
my understanding is that isInBeta only works on ios? but I'm not certain

Reply:
I also thought this might be the case - I'm just getting my emulators working again to double check

Reply:
The beta switch does seem to work on Android in an emulator - I used the same function as what powers the bug button

Reply:
`isInBeta()` will work on android provided we make a different build for beta and prod (which we currently do)